app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt

```diff
@@ -0,0 +1,29 @@
+package ar.com.wolox.android.example.ui.login
+
+import android.util.Patterns
+import ar.com.wolox.android.example.utils.UserSession
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+class LoginPresenter @Inject constructor(private val userSession: UserSession) : BasePresenter<LoginView>() {
+
+    fun onLoginButtonClicked(user: String, password: String) {
+        if (user.isNotEmpty() && password.isNotEmpty()) {
```

Comment:
The card was intended to make you show a different error message for each field 👀 Can you adapt this logic to that?